tests/sys/conn_can/main.c

```diff
@@ -145,25 +147,28 @@ static int _send(int argc, char **argv, bool rtr)
 
     if (rtr) {
         frame.can_id = CAN_RTR_FLAG | strtoul(argv[3], NULL, 16);
-        frame.can_dlc = strtoul(argv[4], NULL, 10);
+        frame.len = strtoul(argv[4], NULL, 10);
     } else {
         frame.can_id = strtoul(argv[3], NULL, 16);
-        frame.can_dlc = argc - 4;
+        frame.len = argc - 4;
     }
-    if (frame.can_dlc > 8) {
+    if (frame.len > DEFAULT_CAN_MAX_DLEN) {
         puts("Invalid length");
         return 1;
     }
 
     if (rtr) {
-        for (int i = 0; i < frame.can_dlc; i++) {
+        for (int i = 0; i < frame.len; i++) {
             frame.data[i] = 0x0;
         }
     } else {
-        for (int i = 0; i < frame.can_dlc; i++) {
+        for (int i = 0; i < frame.len; i++) {
             frame.data[i] = strtol(argv[4 + i], NULL, 16);
         }
     }
+#ifdef MODULE_FDCAN
+    frame.flags |= CANFD_BRS | CANFD_FDF;
```

Comment:
this test should be able to send non FD frames even if the hardware supports fd frames

Reply:
on a short look 44.3.2 of the manual seems to me like enabling these bits will transfer the frame in fd mode -> this function only sends fd frames if the module_fdcan is used

Reply:
maybe add a new function that sends fd frames and let the old one send can-classic frames

Reply:
eg `test_can send ` `test_can fdsend`  they can both be hadled by the send function which decides by string matching whether to set or not set the fd flags

Reply:
done.:)